it/lessons/basics/pattern-matching.md

```diff
@@ -9,7 +9,7 @@ _Pattern matching_ è un aspetto fondamentale di Elixir, permette di abbinare se
 
 ## Operatore match
 
-Se pronto ad essere spiazzato? In Elixir, l'operatore `=` è in realtà un operatore match. Tramite l'operatore match possiamo assegnare ed abbinare valori, diamo uno sguardo:
+Se pronto ad essere spiazzato? In Elixir, l'operatore `=` è in realtà un operatore match. Tramite l'operatore match possiamo assegnare ed abbinare valori, simile al segno uguale in algebra. Scrivendolo, transforma l'intera espressione in un'equazione ed Elixir deve abbinare i valori sulla sinistra a quelli della destra. Se l'operazione ha successo, Elixir ritonerà di valori dell'equazione, altrimenti, genera un errore. Diamo uno sguardo:
```

Comment:
`Elixir ritonerà di valori dell'equazione, altrimenti, genera un errore`

There are two small issues here:

* `ritonerà di valori` should be `ritonerà i valori`
* `ritornerà` (future) Vs `genera` (present) -> it might be more appropriate to pick one form and stick with it. So you might have `ritornerà ... genererà` or `ritorna ... genera`. Not very important of course ;-)

Reply:
I think it's better if we stick to the present form. Made the according changes in efe71e5a7017e4a5602d069b368c9c56d9880e4b